### PR TITLE
feat: nutanix node reboot

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8493
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/controller.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  reboot() {
+    this.isRebooting = true;
+    return this.NutanixNode.reboot(this.nodeId)
+      .then((task) => {
+        this.$rootScope.$broadcast('nutanix.node.reboot', task);
+        this.goBack(
+          this.$translate.instant('nutanix_node_reboot_success', {
+            nodeName: this.node.displayName,
+          }),
+          'DONE',
```
Comment (varun257) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/controller.js:23]: There seems to be a constant `STATUS_DONE`. Can this be used here as well ?

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "nutanix_node_reboot_title": "Redémarrer le serveur",
+  "nutanix_node_reboot_description": "Etes vous sûr de vouloir redémarrer votre serveur {{nodeName}} ?",
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/translations/Messages_fr_FR.json:3]:
```suggestion
  "nutanix_node_reboot_description": "Ếtes vous sûr(e) de vouloir redémarrer votre serveur {{nodeName}} ?",
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  "nutanix_node_reboot_success": "Votre serveur {{nodeName}} va redémarrer.",
+  "nutanix_node_reboot_error": "Une erreur est survenue lors de la demande de redémarrage: {{message}}",
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/translations/Messages_fr_FR.json:5]:
```suggestion
  "nutanix_node_reboot_error": "Une erreur est survenue lors de la demande de redémarrage : {{message}}",
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/translations/Messages_fr_FR.json (hunk 2 above)
Comment (radireddy) [reply packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/reboot/translations/Messages_fr_FR.json:3]: 👍

**Pull request merged**